package com.app.exception;

public class Test2 {

	
	public static void main(String[] args) {
		

		/*ArrayIndexOutofBoundsException
		int x[] = new int[10];
		System.out.println(x[9]);
		//System.out.println(x[15]);
		System.out.println(x[-15]);*/
		
		/*NullPointerException
		String s = null;
		System.out.println(s.length());*/
		
		/*ClassCastException
		//String s = new String("ashok");
		Object o = new Object();
		String s = (String)o;
		System.out.println(s);*/
		
		/*IllegalArgumentException
		Thread t = new Thread();
		t.setPriority(10);
		t.setPriority(100);*/
		
		/*NumberFormatException
		int i = Integer.parseInt("10");
		int j = Integer.parseInt("ten");*/
		
		
		
	}

}
